chatlearn/models/vllm_module_v2.py

```diff
@@ -101,10 +101,16 @@ def setup_vllm(self, workers):
         else:
             model_loader_extra_config = None
 
+        if self.model_args.get("apply_replica_id_to_seed", True):
+            seed = self.model_args.get("seed", 0) + self.replica_id
```

Comment:
Will this behavior cause the mis-alignment generation result between pure vLLM and us?

Reply:
mis-alignment is unaviodable. In alignment training e2e tasks, we need to generate different outputs for the same several prompts.